libs/clients/zendesk/src/lib/zendesk.config.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from 'zod'
+import { defineConfig } from '@island.is/nest/config'
+
+const schema = z.object({
+  subdomain: z.string(),
+  formEmail: z.string(),
+  formToken: z.string(),
+})
+
+export const ZendeskServiceConfig = defineConfig<z.infer<typeof schema>>({
+  name: 'ZendeskServiceConfigurations',
+  schema,
+  load: (env) => ({
+    subdomain: env.required('ZENDESK_CONTACT_FORM_SUBDOMAIN', 'digitaliceland'),
+    formToken: env.required('ZENDESK_CONTACT_FORM_TOKEN', 'undefined'),
```

Comment:
Should this be a string undefined?

Reply:
It needs to be initialised as some value.. I put it as undefined so it will be noticeable that it is not set.